plugins/modules/zos_data_set.py

```diff
@@ -247,6 +247,18 @@
         that is not available, then the value C(TMPHLQ) is used.
     required: false
     type: str
+  force:
```

Comment:
I think we might want to mention the usage of state and force in the state doc, eg there in the doc they describe how the two together behave 

```If state=present and replace=True .....```

Where we might want:
```
If state=absent and force=True, force will only apply to data set members and the data set will be opened with I(DISP=SHR) such that the entire data set can be accessed by other processes while the member is deleted. 
```

Reply:
good point, i'm modifying your suggestion to "if `type =member` ..." rather than having "only apply to data set members" -- i think that's more explicit and quicker to read/understand